### ISSUE TITLE
ImageClassification scoring use ResNet ONNX

### ISSUE THREAD

**Issue by huanbd** [dotnet/machinelearning-samples#732]
I created a sample image classification use ResNet ONNX  like [sample image classification use Inception Tensorflow model](https://github.com/dotnet/machinelearning-samples/tree/master/samples/csharp/getting-started/DeepLearning_ImageClassification_TensorFlow)
My env:
- Microsoft.ML (1.4.0-preview2)
- Microsoft.ML.ImageAnalytics (1.4.0-preview2)
- Microsoft.ML.OnnxTransformer (1.4.0-preview2)
My model ONNX: 
- ResNet version 2 : [ResNet-152](https://github.com/onnx/models/tree/master/vision/classification/resnet)

That is my step:
1/ Use Netron to view shape of model, get input and output, [view this image](https://github.com/huanbd/ImageClassification_ONNX_Score/blob/master/netron_resnet125v2.png)
- input: **data**
- output: **resnetv27_dense0_fwd**
- image size: 224 x 224

2/ Load model use `mlContext.Transforms.ApplyOnnxModel`
3/ Get predictor from `ITransformer`, result is an array float size 1000
4/ Use Softmax(result)
5/ Get BestLabel

This is my sample https://github.com/huanbd/ImageClassification_ONNX_Score
This is my result https://github.com/huanbd/ImageClassification_ONNX_Score/blob/master/result.PNG

But I can't find the right result. Can you help me to review that. where is wrong? Thank you.

**Comment by ashbhandare** [comment]
Could you elaborate on what you mean by the right result? From the screenshot of your result, looks like you are able to get the best label for the 3 images.

**Comment by huanbd** [comment]
Hi @ashbhandare, Thanks for your response! 
It's run but I think something wrong happened because with the same images in a class it's return different result in each image. As you see in [my result](https://github.com/huanbd/ImageClassification_ONNX_Score/blob/master/result.PNG) a.jpg, b.jpg, c.jpg are in class Tench, tinca tinca , but the predict result is wrong. It's happened with all data sample I've tested, So I think I wrong implemented somewhere that I don't know

**Comment by IveJ** [comment]
Hi Issuer,

bc of u utilize mlnet v1.4, ResNet v2, it uses pre-activation, The
transformation should preferrably happen at preprocessing.
https://github.com/onnx/models/tree/master/vision/classification/resnet

**Comment by huanbd** [comment]
Hi @IveJ . Thanks for your response, It helped me better understand about ResNet
